### PR TITLE
Enhancement : API-Gateway Put Integration Response - Adding support f…

### PR DESCRIPTION
**Pull request by usmangani1**
API-Gateway Put Integration Response - Adding support for contentHandling.

https://github.com/localstack/localstack/issues/2273

### CODE REVIEW

**Comment by usmangani1** [comment]
> API-Gateway Put Integration Response - Adding support for contentHandling.
> 
> [localstack/localstack#2273](https://github.com/localstack/localstack/issues/2273)

@bblommers  Need your help in getting this small enhancement merged,

File: moto/apigateway/responses.py [hunk 1]
```diff
@@ -387,13 +387,15 @@ def integration_responses(self, request, full_url, headers):
             elif self.method == "PUT":
                 selection_pattern = self._get_param("selectionPattern")
                 response_templates = self._get_param("responseTemplates")
+                content_handling = self._get_param("contentHandling")
```
Comment (bblommers) [inline moto/apigateway/responses.py:390]: According to [the docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/apigateway.html#APIGateway.Client.put_integration_response), this param is not mandatory

File: tests/test_apigateway/test_apigateway.py [hunk 2]
```diff
@@ -543,7 +543,9 @@ def test_integration_response():
         statusCode="200",
         selectionPattern="foobar",
         responseTemplates={},
+        contentHandling="CONVERT_TO_BINARY"
```
Comment (bblommers) [inline tests/test_apigateway/test_apigateway.py:546]: Can you assert that the param can be retrieved as well? (In one of the tests - doesn't have to be this particular one.)

Also, please remove the contentHandling-param for some of the tests, to verify it's not mandatory.

**Comment by bblommers** [comment]
This is now available in moto >= 1.3.15.dev888

**Pull request merged**